index.bs

```diff
@@ -4687,15 +4687,45 @@ Note: If an error name is not listed here, please file a bug as indicated at the
         <tbody>
             <tr>
                 <td>
-                    "<dfn id="invalidaccesserror" exception export><code>InvalidAccessError</code></dfn>"<br>
-                    <dfn id="dom-domexception-invalid_access_err" for="DOMException" const export><code>INVALID_ACCESS_ERR</code></dfn>&nbsp;(15)
+                    "<dfn id="domstringsizeerror" exception><code>DOMStringSizeError</code></dfn>"<br>
+                    <dfn id="dom-domexception-domstring_size_err" for="DOMException" const><code>DOMSTRING_SIZE_ERR</code></dfn>&nbsp;(2)
+                </td>
+                <td>Use <emu-val>RangeError</emu-val> instead.</td>
+            </tr>
+            <tr>
+                <td>
+                    "<dfn id="nodataallowederror" exception><code>NoDataAllowedError</code></dfn>"<br>
+                    <dfn id="dom-domexception-no_data_allowed_err" for="DOMException" const><code>NO_DATA_ALLOWED_ERR</code></dfn>&nbsp;(6)
+                </td>
+                <td>—</td>
+            </tr>
+            <tr>
+                <td>
+                    "<dfn id="invalidaccesserror" exception><code>InvalidAccessError</code></dfn>"<br>
+                    <dfn id="dom-domexception-invalid_access_err" for="DOMException" const><code>INVALID_ACCESS_ERR</code></dfn>&nbsp;(15)
                 </td>
                 <td>
                     Use <emu-val>TypeError</emu-val> for invalid arguments,
                     "{{NotSupportedError!!exception}}" for unsupported operations, and
                     "{{NotAllowedError!!exception}}" for denied requests.
                 </td>
             </tr>
+            <tr>
+                <td>
+                    "<dfn id="validationerror" exception><code>ValidationError</code></dfn>"<br>
+                    <dfn id="dom-domexception-validation_err" for="DOMException" const><code>VALIDATION_ERR</code></dfn>&nbsp;(16)
+                </td>
+                <td>—</td>
+            </tr>
+            <tr>
+                <td>
+                    "<dfn id="typemismatcherror" exception><code>TypeMismatchError</code></dfn>"<br>
+                    <dfn id="dom-domexception-type_mismatch_err" for="DOMException" const><code>TYPE_MISMATCH_ERR</code></dfn>&nbsp;(17)
+                </td>
+                <td>
+                    Use <emu-val>TypeError</emu-val> instead.
+                </td>
```

Comment:
Should we maybe give this row a class and a slightly different background color to indicate it is not to be used?